external-crates/move/crates/move-package/src/resolution/dependency_graph.rs

```diff
@@ -289,14 +318,6 @@ impl<Progress: Write> DependencyGraphBuilder<Progress> {
         }
 
         // collect sub-graphs for "regular" and "dev" dependencies
-        let root_pkg_id = custom_resolve_pkg_id(&root_manifest).with_context(|| {
```

Comment:
Not sure I understand this change

Reply:
`root_pkg_id` is needed earlier in the code to allow (earlier) read from the lock file